internal/log_analysis/log_processor/parsers/numerics/numerics.go

```diff
@@ -0,0 +1,49 @@
+package numerics
+
+/**
+ * Panther is a scalable, powerful, cloud-native SIEM written in Golang/React.
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import (
+	"strconv"
+	"strings"
+)
+
+// this is an int that is read from JSON as either a string or int
+type Integer int
+
+func (i *Integer) String() string {
+	if i == nil {
+		return "nil"
+	}
+	return strconv.Itoa((int)(*i))
+}
+
+func (i *Integer) MarshalJSON() ([]byte, error) {
+	return ([]byte)(i.String()), nil
+}
+
+func (i *Integer) UnmarshalJSON(jsonBytes []byte) (err error) {
+	cleanBytes := strings.Replace((string)(jsonBytes), `"`, "", -1) // remove quotes
```

Comment:
We just need to remove quotes from either side, right?

```suggestion
	trimmed := strings.Trim(string(jsonBytes), `"`))
```

Reply:
Done